### DISCUSSION TITLE
State after 2.7 compared to orca?

### DISCUSSION THREAD

**Discussion by FelixConni** [supermerill/SuperSlicer#4134] [General]
Hey,
I loved ss all the time.
I stopped using ss when it didn't got updated a while ago and then switched to prusa and then to orca and now im happy with orca. But Not sure if i should switch back after 2.7 do you have all the features of orca implemented in ss? if not when would this be done ish? 
Also maybe a question why someone should use ss over any other (like orca) or not?

**Comment by darco415** [comment]
2.7 isnt out yet....

**Reply by FelixConni** [reply]
I know but he knows everything he changed and designed and even future plans. The idea is to understand the creator so I can understand the creation. so I can give recommendations to my friends and share knowledge.

**Reply by supermerill** [reply]
I want to create the most wonderful software of the world.
But you know, the brick wall named reality is always in the way.

I'll merge with prusa 2.7
Then I have some pull request to merge, and some features also (like perimeter count split for contour/hole)
Then we'll see what's important to get from orca that prusa ans superslicer still don't have.
